internal/kube/client.go

```diff
@@ -53,7 +53,7 @@ func NewFromConfigWithTimeout(url, file string, t time.Duration) (KymaKube, erro
 		return nil, err
 	}
 
-	octClient, err := octopus.NewOctopusRESTClient(2 * time.Second)
+	octClient, err := octopus.NewFromConfig(config)
```

Comment:
Do we have a default timeout when creating a client in this way?

Reply:
Yes, the default timeout is 30 seconds.
See: https://github.com/kyma-project/cli/blob/e6ae0cc6e8bb7cf8262fac15682a806e519cc609/internal/kube/client.go#L32

Reply:
Thanks. Would you consider to keep the timeout within 2-5 sec range?

Reply:
30 sec is just a reasonable default, but of course different commands will have different needs.
That is why each command can specify a custom timeout when creating the client with `NewFromConfigWithTimeout()`.
If you think a command needs a 2sec timeout then just create the clientset with that timeout.